Created term entries and made other edits re: Python Sets (#711)
Codecademy/docs: content/python/concepts/built-in-functions/terms/len/len.md

---
Title: 'len()'
Description: 'Returns the length of an object, which can either be a sequence or collection.'
Subjects:
  - 'Computer Science'
  - 'Data Science'
Tags:
  - 'Functions'
  - 'Methods'
  - 'Strings'
CatalogContent:
  - 'learn-python-3'
  - 'paths/computer-science'
---

The built-in `len()` function returns the length of an object, which can either be a sequence or collection.

## Syntax

```pseudo
len(object)
```

> Again, any `object` passed to the `len()` function is commonly one of the following:

Reviewer: ```suggestion
An `object` passed to the `len()` function is commonly one of the following:
```

- It can be a sequence such as a [string](https://www.codecademy.com/resources/docs/python/strings) or [tuple](https://www.codecademy.com/resources/docs/python/tuples).
- It can also be a collection such as a [dictionary](https://www.codecademy.com/resources/docs/python/dictionaries), or [set](https://www.codecademy.com/resources/docs/python/set).

## Example

The example below demonstrates how the `len()` function is used in a Python program:

```py
print(len("Hello, World!"))
# Output: 13
```

## Codebyte Example

> In the example below, the `len()` function is used to return the length of a few collections, including a [list](https://www.codecademy.com/resources/docs/python/lists):

Reviewer 2: ```suggestion
In the example below, the `len()` function is used to return the length of a string, dictionary, and [list](https://www.codecademy.com/resources/docs/python/lists):
```
